### ISSUE TITLE
MZmine MS/MS

### ISSUE THREAD

**Issue by BrandiWest** [mzmine/mzmine2#67]
So I've been playing around with MZmine and realized that there aren't any good tutorials or examples of how to use the more advance features.  The only manual I was able to find was for v2.3 which doesn't seem to help to much.  I'm trying to use my MS2 data to help with identification but I have no idea how to use it (other than detecting the masses and making a mass list).  I think I need to use the MS/MS builder but I have no idea how to get the information I want or how to use it for future steps once I have it.

Any help would be appreciated.

Thanks!

Brandi

**Comment by tomas-pluskal** [comment]
Hi Brandi,

Each MZmine version comes with a manual (PDF) and each module has a build-in help that describes its details. If you feel the documentation could be improved somewhere (I'm sure it could), please send us your improved version and we will readily add it to MZmine.

Closing this issue.

**Comment by Biljana-ruc** [comment]
Hi 

I have isntalled the newest version and have problems with getting correct identification when i import an adduct file i made it in excel and save it as cvs file. I saw one video tutorial that says that the adduct file is made by MS/MS and i have gone through all the possibilitites and can not simply find a way how to do the adduct lsit in the proggram. Any help here?

Also, maybe a stupid question, but how does the program knows if the adduct observed is from the mass of interest? Lets say my mass is 1478 and then there is a peak of 1501 and then the 1478 is identified, but the M+Na adduct is not. 

I appreciate your time and help. 

Many thanks

Bibi
